pkg/dnshelpers/util.go

```diff
@@ -0,0 +1,135 @@
+package dnshelpers
+
+import (
```

Comment:
It seems this package is completely unused by the PR?

Reply:
Huh! I must have changed it to not be used somewhere.  It was originally..  Thank you :)